data/campaigns/Winds_of_Fate/scenarios/01_The_Hunt.cfg

```diff
@@ -211,27 +154,29 @@
             time=1500
         [/delay]
         [message]
-            speaker=Reshan
+            speaker=Resha
             message= _ "Behold, the Isle of Gaall.
 The most perilous hunting ground in the Domain of Vladnir.
-Many a hunter has become its prey."
+
+Spurred by the hunger cries of their hatchlings, many a hunter has journeyed to this place.
+Only to become its prey."
         [/message]
         [message]
             speaker=Gorlack
             message= _ "Our trial for aspirancy has come at last.
 The hunt that awaits us below will prove a worthy test."
         [/message]
         [message]
-            speaker=Reshan
+            speaker=Resha
             message= _ "If only we could still count Karron among us...
 
-I fear without his aid this trial will outmatch us.
+I fear without her aid this trial will outmatch us.
```

Comment:
Are we absolutely sure we want to make the flying fire lizards exhibit human sexual dimorphism? I would have thought having them be hermaphroditic would be more interesting, but I'd like to hear some other peoples' reasoning too.

Reply:
This is actually reptilian, avian and mammalian sexual dimorphism, not just human. The woses would be the best fit if you want a hermaphroditic species (as such is very common in the plant kingdom).

Reply:
Good point I totally forgot my bio classes.

Reply:
I already assumed the woses were hermaphroditic.

Hermaphroditism is pretty common even among animals (about 5%, apparently). I'm not saying the drakes should be hermaphroditic, but it wouldn't be _that_ strange if they were.

Can someone explain exactly where sexual dimorphism is being exhibited here? Nearly all the text changes are just pronoun changes… I don't see anything anywhere to suggest sexual dimorphism.

(Note: In case someone is confused, "sexual dimorphism" is when the males and females look dramatically different. A good example is ducks. So maybe that wasn't actually what you meant?)